kolibri/core/assets/src/core-app/mediator.js

```diff
@@ -82,13 +93,16 @@ module.exports = class Mediator {
     this._executeCallbackBuffer(kolibriModule);
     logging.info(`KolibriModule: ${kolibriModule.name} registered`);
     this.emit('kolibri_register', kolibriModule);
-    if (this._ready) {
-      kolibriModule.ready();
-    } else {
-      this._eventDispatcher.$once('ready', () => {
+    const ready = () => {
```

Comment:
No, literally just nesting the original code block inside a function, so that it can be used as a callback from a Promise (or called independently to keep it DRY).